options/options.h

```diff
@@ -144,6 +146,7 @@ class PonoOptions
   static const bool default_cegp_axiom_red_ = true;
   static const std::string default_profiling_log_filename_;
   static const bool default_mod_init_prop_ = false;
+  static const bool default_assume_prop_ = false;
```

Comment:
how about turning it on by default.

Reply:
Yeah I'm okay with that! Do you think we could run into any unsoundness issues by accident? Or as long as it's safety checking only we should always be good?

Reply:
I think that we are good for invariant properties and that's what we support